.claude/agents/meta-worker.md

```
@@ -0,0 +1,125 @@
+Store the root path — you'll need it for every file operation.
+
+**Dependency resolution**: When you specify `--repo meta_git_cli`, meta automatically includes transitive dependencies defined in `.meta.yaml` (`provides`/`depends_on`). Use `--no-deps` only if you need precise control.
+
+### Phase 2: Enter Worktree
+
+```bash
+cd <root>
+```
+
+From here, all `meta` commands automatically scope to worktree repos. Verify with:
+```bash
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)